gooddata-sdk/gooddata_sdk/catalog/base.py

```diff
@@ -65,3 +70,7 @@ def client_class() -> Any:
     def to_api(self) -> Any:
         dictionary = self._get_snake_dict()
         return self.client_class().from_dict(dictionary, camel_case=False)
```

Comment:
Not part of this PR, but `client_class` should return Type which is bound to Base class.

Could be `def client_class() -> Any:` replaced with `def client_class() -> Type[T]:` ?

Reply:
I do not think it should. Client class returns the related class of the client, which is unrelated to the Base class.

Reply:
It actually returns Type[ModelNormal], but there is an issue that there is a ModelNormal for each client.